src/ViewModel/Converter/ArticleTeaserConverter.php

```diff
@@ -34,6 +39,20 @@ public function convert($object, string $viewModel = null, array $context = [])
             $image = null;
         }
 
+        $formats = [];
+
+        if ($object instanceof ArticleVoR || null === $object->getPdf()) {
```

Comment:
isn't the second condition redundant? All HTML versions come from VOR?

Seems impossible in both practice and theory to have a POA with a RDS companion, if it makes any difference (don't thiink so).

Reply:
`ArticlePoA::getPdf()` should probably be updated to say that it will definitely return a string. But yeah, let's remove it. Safe to produce an empty list.